src/api/routes/ws-rpc.ts

```diff
@@ -78,23 +81,23 @@ export function createWsRpcRouter(db: DataStore, server: http.Server): WebSocket
       let rpcReqs = Array.isArray(parsedRpcReq) ? parsedRpcReq : [parsedRpcReq];
 
       // Ignore client notifications, spec dictates server should never respond to these.
-      rpcReqs = rpcReqs.filter(req => req.type !== RpcStatusType.notification);
+      rpcReqs = rpcReqs.filter(req => req.type !== 'notification');
 
       const responses: JsonRpc[] = rpcReqs.map(rpcReq => {
         switch (rpcReq.type) {
-          case RpcStatusType.request:
+          case 'request':
```

Comment:
Why killing the string enum?

Reply:
It was throwing with `RpcStatusType` being undefined. I'm not sure how it ever worked. Maybe ts-node doesn't compile the const enum when debugging now, not really sure.